### ISSUE TITLE
How to add cuts/constraints to Conic Benchmark Library (CBLIB) Instances?

### ISSUE THREAD

**Issue by miladdf94** [jump-dev/MathOptInterface.jl#1107]
Hi everyone
@blegat @ccoffrin @chriscoey @odow @mlubin 

I am new to Julia and need some help with Julia/Mosek.
I have used the following code from jump-dev/MathOptInterface.jl#950 to read Conic Benchmark Library (CBLIB) instances.
However, I need to add some constraints/cuts to these instances and re-run the optimization problem using Mosek.

I need help with adding cuts/constraints to the instance. For example, let's say we want to add this cut:
x1 + x5 <= 7

I also need help with building CBF instances in Julia and obtaining the constraint matrix of these instances. I tried to use `ConicBenchmarkUtilities` to convert between CBF and `MathProgBase` conic format. But it seems `MathProgBase` is deprecated and the code does not work.

The code that I used is as follows (jump-dev/MathOptInterface.jl#950)

```julia
using MathOptInterface, MathOptFormat, MosekTools

const MOI = MathOptInterface

filename = "instances/10_0_1_w.cbf"

mathoptformat_model = MathOptFormat.CBF.Model()

solver = Mosek.Optimizer()

#user_model = nothing
user_model = MOI.Bridges.full_bridge_optimizer(solver , Float64)

println("check 1")
MOI.read_from_file(mathoptformat_model, filename)
println("check 2")
MOI.copy_to(user_model, mathoptformat_model)
@show user_model
println("check 3")
MOI.optimize!(user_model)
x = MOI.get(user_model, MOI.VariableIndex, "x")
@show x

**Comment by 1ozturkbe** [comment]
@miladdf94 you can get the constraint matrix with the following code:
``` julia
    using ConicBenchmarkUtilities
    dat = readcbfdata(filename);
    c, A, b, constr_cones, var_cones, vartypes, sense, objoffset = cbftompb(dat);
```
You can find it also find it [here](https://github.com/JuliaOpt/ConicBenchmarkUtilities.jl).

**Comment by 1ozturkbe** [comment]
It is irritating however that there is not a 'bridgeless' integration between MOI and/or JuMP and forms of problem data storage. I am also wondering how to add cuts/constraints to the underlying model in presence of .Bridges.

**Comment by odow** [comment]
I haven't tried, because I don't have the cbf file, but I imagine you can do something like this:
```Julia
using JuMP, MosekTools

model = read_from_file("instances/10_0_1_w.cbf")
set_optimizer(model, Mosek.Optimizer)
x1 = variable_by_name(model, "x1")
x5 = variable_by_name(model, "x5")
@constraint(model, x1 + x5 <= 7)
optimize!(model)
```

**Comment by odow** [comment]
> It is inconvenient however that there is not a 'bridgeless' integration between MOI and/or JuMP and forms of problem data storage.

See https://jump.dev/JuMP.jl/v0.21.1/solvers/#JuMP.direct_model
```Julia
model = direct_model(Mosek.Optimizer())
```

**Comment by miladdf94** [comment]
@1ozturkbe 
I was able to get the constraint matrix using the mentioned code. Thanks.

I have another issue though.
I want to get the coefficients of each variable in a set of constraints, for example less_than_constraints.

Here is the code I used:
`less_than_constraints = all_constraints(model, GenericAffExpr{Float64,VariableRef}, MOI.LessThan{Float64})
con = constraint_object(less_than_constraints[1])
con.func`

It gives me the constraint function as:
`3.3x[1] + 2x[2]`.

How can I store the coefficients of x[1] and x[2], i.e., 3.3 and 2? I need to get the coefficients of the mentioned constraint function, not the whole function.

Thanks in advance.
